lms/djangoapps/courseware/views/index.py

```diff
@@ -406,6 +406,7 @@ def _create_courseware_context(self):
                 courseware_context['entrance_exam_passed'] = user_has_passed_entrance_exam(self.request, self.course)
 
         # staff masquerading data
+        courseware_context['real_user'] = self.real_user
```

Comment:
Could this be moved to the block initializing `courseware_context` with most of its contents?